### PR TITLE
Milestone 1: Building the Mobile skeleton of the personal Portfolio website by following Figma specifications to meet the predefined requirements

### PR DESCRIPTION
**Pull request by jodrack07**
Throughout this milestone, I implement a mobile version skeleton of a Portfolio website where I applied some properties related to the Flexbox layout to properly and easily arrange items in a box.

During this project building I have used these tools :
1. HTML, for Markup✅
2. CSS for Styling where I applied Flexbox layout properties✅
3. Figma for UI reference✅
4. Linters for source code checkup✅

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
+    <title>Jodrack</title>
+</head>
+<body>
+    <header class="header">
+        <nav class="navbar">
+          <div class="logo"><h1>My Logo</h1></div>
```
Comment (Deepakdanger) [inline index.html]: - [ ] `My Logo` should be inside `a` tag. Kindly remove `h1` tag from here.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            <i class="fa fa-bars"></i>
+          </div>
+        </nav>
+    </header>
+
+    <main class="main">
+        <section class="showcase">
+          <div class="showcase__presentation">
+            <h1 class="presentation">I’m Jodrack</h1>
+            <h1 class="sub__presentation">Glad to see you!</h1>
```
Comment (Deepakdanger) [inline index.html]: - [ ] There can be only one `h1` tag on each website page as a title, so let's add `br` tag for breakpoint rather using the `h1` tag on both lines.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @joseph07-drack, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (jodrack07) [reply index.html]: Oh!! thanks so much. I am going to add.

File: index.html (hunk 2 above)
Comment (jodrack07) [reply index.html]: Great. Let me fix it.

**Review by gemmen29 — APPROVED** [review]
Hi @joseph07-drack, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jodrack07** [comment]
Thank you for reviewing and validating my work.

**Pull request merged**